shanoir-ng-front/src/app/welcome/welcome.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { WelcomeComponent } from './welcome.component';
+
+describe('WelcomeComponent', () => {
+  let component: WelcomeComponent;
+  let fixture: ComponentFixture<WelcomeComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ WelcomeComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(WelcomeComponent);
```

Comment:
Is all this necessary ?

Reply:
This file is automatically generated by "ng generate component" but could be removed

Reply:
For the moment, we don't really have unit tests in Angular. We can discuss about it to add tests if you want, but in this case we should test it correctly ;)